### ISSUE TITLE
Tilemap save settings

### ISSUE THREAD

**Issue by Banduck** [godotengine/godot#22832]
It would be very useful and time saving if the tilemap & tileset editor settings were saved.
For example:
- Tilemap zoom-level
- Zoom-level of the tileset editor
- Region snap options

Every time you open Godot you have to adjust all these settings again.

**Comment by KoBeWi** [comment]
Sort of related to #24114

**Comment by erictuvesson** [comment]
Is this still relevant?

**Comment by KoBeWi** [comment]
Yes.

**Comment by erictuvesson** [comment]
I am not following on all the points, could they be a little bit more specific?

There is a settings category in `Editor Settings > Editors > Tile Map` right now, and the idea is to add more default options in there?

**Comment by KoBeWi** [comment]
No, things like zoom or scroll position should be saved somewhere in project metadata. E.g. Godot stores view position for each scene, but it's not saved in the scene file, but somewhere in the user directory. Same could be done for TileMap and TileSet Editors.
